charms/istio-pilot/metadata.yaml

```diff
@@ -22,8 +22,9 @@ provides:
     versions: [v1]
   ingress:
     interface: ingress
-    schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/ingress.yaml
-    versions: [v1, v2]
+    #schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/ingress.yaml
+    schema: https://raw.githubusercontent.com/canonical/operator-schemas/1ed74c640bc289b71f261cda67177ee5209a1562/ingress.yaml
```

Comment:
TODO: Remove once https://github.com/canonical/operator-schemas/pull/4 is merged and before this is.